WordPress-Core/ruleset.xml

```diff
@@ -356,6 +356,13 @@
 	<!-- Class opening braces should be on the same line as the statement. -->
 	<rule ref="Generic.Classes.OpeningBraceSameLine"/>
 
+	<!-- References to self in a class should be lower-case and not have extraneous spaces,
+		 per implicit conventions in the core codebase; the NotUsed code refers to using the
+		 fully-qualified class name instead of self, for which there are instances in core. -->
+	<rule ref="Squiz.Classes.SelfMemberReference"/>
+	<rule ref="Squiz.Classes.SelfMemberReference.NotUsed">
+		<severity>0</severity>
```

Comment:
We know WP doesn't follow half of what's in this main ruleset, so why should this exception cater just for WP Core? It would mean that every single other plugin and theme project who wanted to enable the check would have to explicitly set it, having looked up what severity it should be.

Could it not be left enabled, and let a fictional http://develop.svn.wordpress.org/trunk/phpcs.xml.dist disable it?

What are the chances of getting a `phpcs.xml.dist` added to core, that could handle these "WordPress doesn't do it right yet" cases, and let this `ruleset.xml` be the showcase example of how every plugin and theme in the community should be doing it?